tools/gdb/utils.py

```diff
@@ -1,8 +1,6 @@
 ############################################################################
 # tools/gdb/utils.py
 #
```

Comment:
Removed SPDX License Identifier by mistake?

Reply:
It's a mistake. Added back.